VirtualClosetApp/www/js/controllers.js

```diff
@@ -128,7 +128,7 @@ angular.module('starter.controllers', [])
       myCloset.Jackets = "";
       myCloset.Accessories = "";
 
-      myCloset.Connections = "";
+      myCloset.Connections = [];
       myCloset.Wishlist = "";
```

Comment:
Should this change be propagated to the other controllers listed here? It may prevent issues down the line.